mage/query-modules/python/elasticsearch-synchronization.md

```diff
@@ -0,0 +1,315 @@
+---
+id: elasticsearch-synchronization
+title: elasticsearch_synchronization
+sidebar_label: elasticsearch_synchronization
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
```

Comment:
Why do you need this?

Reply:
Other modules have it so I copied it

Reply:
I have no idea what it does :D let's leave it be :D :D